roles/pulp-status/tasks/main.yml

```diff
@@ -1,11 +1,28 @@
 ---
 
+- name: Set apiVersion and kind variables
+  set_fact:
+    api_version: '{{ hostvars["localhost"]["inventory_file"].split("/")[4:6] | join("/")  }}'
+    kind: '{{ hostvars["localhost"]["inventory_file"].split("/")[6]  }}'
+
+- name: Update migrantDatabaseConfigurationSecret status
+  operator_sdk.util.k8s_status:
+    api_version: '{{ api_version }}'
+    kind: "{{ kind }}"
```

Comment:
not sure how debuggable `k8s_status` is, I'm wondering if we should add a `debug` for `api_version` and `kind`